### DISCUSSION TITLE
Report for the diff files only

### DISCUSSION THREAD

**Discussion by trOnk12** [PatilShreyas/compose-report-to-html#89] [Ideas]
Is it possible to create a report only for the Composables that are part of the diff of the PR ?

**Comment by trOnk12** [comment]
@PatilShreyas  any feedback ?

**Reply by PatilShreyas** [reply]
It's actually a good idea. But currently it's not possible to do with this. Since to calculate this, we'll require a base report with all details and a current report to compare. So in order to have this feature, will have to create something centralized
